minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java

```diff
@@ -90,9 +96,86 @@ public void start(TachyonConf conf) throws IOException, ConnectionFailedExceptio
     CommonUtils.sleepMs(10);
```

Comment:
I've split the waiting into `waitForWorkerReady()` and `waitForMasterReady()` so that we can avoid that sleep as well